install/kubernetes/helm/istio/values-istio-minimal.yaml

```diff
@@ -0,0 +1,47 @@
+#
+# Minimal Istio Configuration: https://istio.io/docs/setup/kubernetes/minimal-install/
+#
+pilot:
+  enabled: true
+  sidecar: false
+  
+gateways:
+  enabled: false
+
+security:
+  enabled: false
+
+sidecarInjectorWebhook:
+  enabled: false
+
+galley:
+  enabled: false
+
+mixer:
```

Comment:
shouldn't this be `mixer.pilot=false, mixer.telemetry=false`?

Reply:
yes, my bad, cp and paste.  thanks!